### ISSUE TITLE
fluid sph

### ISSUE THREAD

**Issue by scipioni** [DanielChappuis/reactphysics3d#5]
Are you planning some fluid or particles simulation ?

**Comment by DanielChappuis** [comment]
Yes it is planned but not in the next release (version 0.6.0). Maybe in the following one.
